test/services/return-requirements/add.service.test.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../support/database.js')
+const SessionHelper = require('../../support/helpers/session.helper.js')
+
+// Thing under test
+const AddService = require('../../../app/services/return-requirements/add.service.js')
+
+describe('Return Requirements - Add service', () => {
+  let session
+
+  beforeEach(async () => {
+    await DatabaseSupport.clean()
+
+    session = await SessionHelper.add({
+      data: {
+        checkPageVisited: false,
+        licence: {
+          id: '8b7f78ba-f3ad-4cb6-a058-78abc4d1383d',
+          currentVersionStartDate: '2023-01-01T00:00:00.000Z',
+          endDate: null,
+          licenceRef: '01/ABC',
+          licenceHolder: 'Turbo Kid',
+          startDate: '2022-04-01T00:00:00.000Z'
+        },
+        journey: 'returns-required',
+        requirements: [{}],
+        startDateOptions: 'licenceStartDate',
+        reason: 'major-change'
+      }
+    })
+  })
+
+  describe('when called', () => {
+    it('adds another empty object to the requirement array in the current setup session record', async () => {
+      await AddService.go(session.id)
+
+      const refreshedSession = await session.$query()
+
+      expect(refreshedSession.data.requirements.length).to.equal(2)
+      expect(refreshedSession.data.requirements).to.equal([{}, {}])
+    })
+  })
```

Comment:
We're not testing that the service also returns the much needed requirement index. Would you mind adding something for this as well?